dmriprep/__about__.py

```diff
@@ -14,3 +14,5 @@
 DOWNLOAD_URL = (
     'https://github.com/nipreps/{name}/archive/{ver}.tar.gz'.format(
         name=__packagename__, ver=__version__))
+
+__ga_id__ = "UA-156165436-1"
```

Comment:
I'd keep this here, but in the long term, I'd like to think of a better config handling. Some kind of singleton pattern on a conf(ig) module.